README.md

```diff
@@ -14,4 +14,14 @@ groundwater/surface water interactions by predicting the movement of the
 water table, which determines where saturated land-surface areas develop
 and have the potential to produce saturation overland flow. TOPMODEL was
 originally developed by
-[Beven & Kirby (1979)](https://www.tandfonline.com/doi/abs/10.1080/02626667909491834).
\ No newline at end of file
+[Beven & Kirby (1979)](https://www.tandfonline.com/doi/abs/10.1080/02626667909491834).
+
+### Links and Documentation
```

Comment:
Suggest more detail in the descriptions next to the links

Reply:
Each link now contains a description; the title of the file itself is rather relevant as well...